NOAA-FIMS/FIMS
# Patch codespace

Thanks for opening a pull request (PR) to FIMS! Please describe the changes under the headings below. Please also go through the checklist below to ensure the appropriate steps have been taken during development. For smaller changes, feel free to skip steps that aren't relevant, but at least describe the feature under the "what is the feature".

# What is the feature?
Fixes the FIMS codespace

# How have you implemented the solution?
Use new codespace features to simplify the Dockerfile and .devcontainer.json file. I removed Intel's MKL for now since we don't have the big var-covar matrices yet that this would matter wrt performance. 

# Does it impact any other area of the project?
No

# Does this change impact the model input or output?*
No

# How to test this change
Open a codespace on this branch and test out the Google tests and devtools::check(). NOTE: you will need to run 
```
options(repos = c(CRAN = "https://cloud.r-project.org/"))
devtools::check()
```

# Developer pre-PR Checklist

Please do these steps locally for big changes. Note GitHub Actions does all of these checks automatically when pushing to the repository. Please see [code development section in the contributor guide](https://noaa-fims.github.io/collaborative_workflow/contributor-guidelines.html#code-development) for details.
